ci/azure-test-docker.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+- job: ${{ parameters.name }}
+  displayName: ${{ parameters.displayName }}
+  pool:
+    vmImage: ubuntu-16.04
+
+  steps:
+  - script: ./integration_test
+    displayName: Run integration test within Docker
+  
+  - script: |
+      docker login -u $(dockerUsername) -p $(dockerPassword)
+      docker push starshipcommand/starship-test
+    # Only push new image if on master and build is passing
+    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    displayName: Push image to dockerhub
```

Comment:
It's just a tiny nitpick ... but can you indent these lines to match up with the above lines 😄.

Reply:
Yep, that indentation is needed.
YAML can be weird-looking. 😛

Reply:
Understand